pymc/tests/test_distributions.py

```diff
@@ -2216,20 +2216,45 @@ def test_multinomial(self, n):
             lambda value, n, p: scipy.stats.multinomial.logpmf(value, n, p),
         )
 
-    def test_multinomial_invalid(self):
-        # Test non-scalar invalid parameters/values
+    def test_multinomial_invalid_value(self):
+        # Test passing non-scalar invalid parameters/values to an otherwise valid Multinomial, 
+        # evaluates to -inf 
```

Comment:
Since we are only using the value once now, we can immediately create the final one (instead of doing `value[1] -= 1` after creation